proprietary/tokens/src/components/amsterdam/button.tokens.json

```diff
@@ -1,6 +1,13 @@
 {
   "amsterdam": {
     "button": {
+      "font-family": { "value": "{amsterdam.typography.font-family}" },
+      "gap": { "value": "{amsterdam.spacing.inset.md}" },
+      "padding-inline-start": { "value": "{amsterdam.spacing.inset.md}" },
+      "padding-inline-end": { "value": "{amsterdam.spacing.inset.md}" },
+      "padding-block-start": { "value": "{amsterdam.spacing.inset.sm}" },
+      "padding-block-end": { "value": "{amsterdam.spacing.inset.sm}" },
```

Comment:
The spacing tokens aren't really well thought out yet, I wouldn't use them here.

Reply:
They were also used in the previous button setup and they're just `1rem` and `.5rem` values.